_data/toc.yaml

```diff
@@ -22,43 +22,49 @@ guides:
   section:
   - path: /engine/installation/
     title: Install Docker
-  - sectiontitle: Docker Edge
+  - sectiontitle: Docker EE
     section:
-    - path: /edge/
-      title: Overview
-    - path: /edge/engine/reference/commandline/docker/
-      title: Docker Edge CLI reference
-      nosync: true
-    - path: /engine/userguide/eng-image/multistage-build/
-      title: Use multi-stage builds (Edge only)
-  - path: /docker-for-mac/install/
-    title: Docker for Mac
-  - path: /docker-for-windows/install/
-    title: Docker for Windows
-  - path: /docker-ee-for-windows/install/
-    title: Docker EE for Windows Server
-  - sectiontitle: Docker for Linux
+    - path: /docker-ee-for-windows/install/
+      title: Docker EE for Windows Server
+    - path: /engine/installation/linux/docker-ee/ubuntu/
+      title: Docker EE for Ubuntu
+    - path: /engine/installation/linux/docker-ee/rhel/
+      title: Docker EE for RHEL
+    - path: /engine/installation/linux/docker-ee/centos/
+      title: Docker EE for CentOS
+    - path: /engine/installation/linux/docker-ee/oracle/
+      title: Docker EE for Oracle Linux
+    - path: /engine/installation/linux/docker-ee/sles/
+      title: Docker EE for SLES
+  - sectiontitle: Docker CE
     section:
-    - path: /engine/installation/linux/ubuntu/
-      title: Ubuntu
-    - path: /engine/installation/linux/rhel/
-      title: Red Hat Enterprise Linux
-    - path: /engine/installation/linux/centos/
-      title: CentOS
-    - path: /engine/installation/linux/fedora/
-      title: Fedora
-    - path: /engine/installation/linux/debian/
-      title: Debian
-    - path: /engine/installation/linux/oracle/
-      title: Oracle Linux
-    - path: /engine/installation/linux/suse/
-      title: SuSE Linux Enterprise
+    - path: /docker-for-mac/install/
+      title: Docker for Mac
+    - path: /docker-for-windows/install/
+      title: Docker for Windows
+    - path: /engine/installation/linux/docker-ce/ubuntu/
+      title: Docker CE for Ubuntu
+    - path: /engine/installation/linux/docker-ce/debian/
+      title: Docker CE for Debian
+    - path: /engine/installation/linux/docker-ce/centos/
+      title: Docker CE for CentOS
+    - path: /engine/installation/linux/docker-ce/fedora/
+      title: Docker CE for Fedora
     - path: /engine/installation/linux/other/
```

Comment:
This URL is broken; https://deploy-preview-3350--docker-docs.netlify.com/engine/installation/linux/other/

Should be

```
/engine/installation/linux/docker-ce/other/
```